packages/core/admin/_internal/node/build.ts

```diff
@@ -79,8 +79,6 @@ const build = async ({ logger, cwd, tsconfig, ignorePrompts, ...options }: Build
   console.log('');
 
   try {
-    EE.init(cwd);
```

Comment:
where was the init moved to ? Is it not necessary to initialize it?

Reply:
It was usefull before to know at build time the EE feature for the FE but nowdays it's managed at runtime so we don't need that anymore. I couldn't fine any other reason to keep it but we will need to test that during our alpha/beta for sure :)